hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -46,6 +49,13 @@
           "false/unspecified indicates otherwise")
   private Boolean isGdprEnforced;
 
+  @CommandLine.Mixin
+  private SetSpaceQuotaOptions quotaOptions;
+
+  @Option(names = {"--key-quota"},
+      description = "Key counts of the newly created bucket (eg. 5)")
+  private long quotaInCounts = OzoneConsts.QUOTA_RESET;
```

Comment:
Similarly to the option for `ClearQuotaHandler`:

by using a bit more generic option name `--count-quota` (and description), this could be moved into `SetSpaceQuotaOptions` (and unified with volume's `--bucket-quota` option).